docs/openstack/backend_services_deployment.md

```diff
@@ -103,40 +104,68 @@ podified OpenStack control plane services.
     secret: osp-secret
     storageClass: local-storage
 
+    cinder:
+      enabled: false
+      template:
+        cinderAPI: {}
+        cinderScheduler: {}
+        cinderBackup: {}
+        cinderVolumes: {}
+
+    dns:
+      enabled: true
+      template:
+        externalEndpoints:
+        - ipAddressPool: ctlplane
+          loadBalancerIPs:
+          - 192.168.122.80
+        options:
+        - key: server
+          values:
+          - 192.168.122.1
+        replicas: 1
+
+    glance:
+      enabled: false
+      template:
+        glanceAPIInternal: {}
+        glanceAPIExternal: {}
+
+    horizon:
+      enabled: false
+      template: {}
+
+    ironic:
+      enabled: false
+      template:
+        ironicConductors: []
+
```

Comment:
I believe we need more updates for network isolation: https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml

Reply:
Yea that's pretty much where i'm taking the info from, but here for the "backend services" section we only include parameters for the services that are enabled. So e.g. network isolation parameters for Keystone went into the `keystone_adoption.md`, and i have WIP updates for the other services that i'll post soon.